client/dashboard/sites/deployments/deployments-list.tsx

```diff
@@ -0,0 +1,45 @@
+import { siteBySlugQuery } from '@automattic/api-queries';
+import { useQuery } from '@tanstack/react-query';
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import { siteRoute } from '../../app/router/sites';
+import { DataViewsCard } from '../../components/dataviews-card';
+import { useDeploymentFields } from './dataviews/fields';
+import { DEFAULT_VIEW, DEFAULT_LAYOUTS } from './dataviews/views';
+import { useAllDeploymentRunsQuery } from './hooks/use-all-deployment-runs-query';
+import type { View } from '@wordpress/dataviews';
+
+export function DeploymentsList() {
+	const { siteSlug } = siteRoute.useParams();
+	const { data: site } = useQuery( siteBySlugQuery( siteSlug ) );
+	const [ view, setView ] = useState< View >( DEFAULT_VIEW );
+
+	const { data: deploymentRuns = [], isLoading } = useAllDeploymentRunsQuery( site?.ID ?? null );
```

Comment:
We showed all code-deployments instead of runs before. If we change to show all runs, then we cannot see all code-deployments. Is it expected? For example, I connected a repo in v1 but didn't run it. However, I cannot see it on v2 since there is no runs for it.